### ISSUE TITLE
Unity 2021.1.9f1 - NullRefException in RegisterCallbacks

### ISSUE THREAD

**Issue by Wobling** [neon-age/Smart-Inspector#6]
Hi,

Just installed, when hovering over the icons in the inspector I get the following errors each time.

```
NullReferenceException: Object reference not set to an instance of an object
AV.Inspector.InspectorEditorTab.<RegisterCallbacks>b__12_1 (UnityEngine.UIElements.MouseEnterEvent evt) (at Assets/Plugins/Smart-Inspector-0.3.0/Base/Elements/InspectorEditorTab.cs:70)
UnityEngine.UIElements.EventCallbackFunctor`1[TEventType].Invoke (UnityEngine.UIElements.EventBase evt) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.EventCallbackRegistry.InvokeCallbacks (UnityEngine.UIElements.EventBase evt) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.CallbackEventHandler.HandleEvent (UnityEngine.UIElements.EventBase evt) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.EventDispatchUtilities.PropagateEvent (UnityEngine.UIElements.EventBase evt) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.MouseEventDispatchingStrategy.SendEventToRegularTarget (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.BaseVisualElementPanel panel) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.MouseEventDispatchingStrategy.SendEventToTarget (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.BaseVisualElementPanel panel) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.MouseEventDispatchingStrategy.DispatchEvent (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel iPanel) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.EventDispatcher.ApplyDispatchingStrategies (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel, System.Boolean imguiEventIsInitiallyUsed) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.EventDispatcher.ProcessEvent (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.EventDispatcher.ProcessEventQueue () (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.EventDispatcher.OpenGate () (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.EventDispatcherGate.Dispose () (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.EventDispatcher.ProcessEvent (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.EventDispatcher.ProcessEventQueue () (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.EventDispatcher.OpenGate () (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.EventDispatcherGate.Dispose () (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.EventDispatcher.ProcessEvent (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.EventDispatcher.Dispatch (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel, UnityEngine.UIElements.DispatchMode dispatchMode) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.BaseVisualElementPanel.SendEvent (UnityEngine.UIElements.EventBase e, UnityEngine.UIElements.DispatchMode dispatchMode) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.UIElementsUtility.DoDispatch (UnityEngine.UIElements.BaseVisualElementPanel panel) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.UIElementsUtility.UnityEngine.UIElements.IUIElementsUtility.ProcessEvent (System.Int32 instanceID, System.IntPtr nativeEventPtr, System.Boolean& eventHandled) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.UIEventRegistration.ProcessEvent (System.Int32 instanceID, System.IntPtr nativeEventPtr) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.UIElements.UIEventRegistration+<>c.<.cctor>b__1_2 (System.Int32 i, System.IntPtr ptr) (at <b3253b03938044f49f49c12d2de4cd71>:0)
UnityEngine.GUIUtility.ProcessEvent (System.Int32 instanceID, System.IntPtr nativeEventPtr, System.Boolean& result) (at <9ff04fda545c4aacb8dab659ad40b2f4>:0)
```

Can reproduce every time.

**Comment by neon-age** [comment]
I think this is already fixed on upcoming v0.5.5 branch, will release soon!

**Comment by Wobling** [comment]
I have updated to v0.6.0 and it appears to be resolved.

Thanks for the quick work!
